### PR TITLE
License update and default value fix (master)

### PR DESCRIPTION
**Pull request by phillip-kruger**
This is a forward-port for #365

Signed-off-by:Phillip Kruger <phillip.kruger@gmail.com>

### CODE REVIEW

File: client/spec/src/main/asciidoc/index.asciidoc [hunk 1]
```diff
@@ -23,16 +23,18 @@
 :version-label!:
 :sectanchors:
 :doctype: book
-:license: Apache License v2.0
+:license: Eclipse Foundation Specification License v1.0
```
Comment (andymc12) [inline client/spec/src/main/asciidoc/index.asciidoc:26]: do we need to change from Apache v2 to Eclipse license?  If so, do we need to also change the headers in the asciidoc files?

File: full/spec/src/main/asciidoc/index.asciidoc [hunk 2]
```diff
@@ -23,18 +23,20 @@
 :version-label!:
 :sectanchors:
 :doctype: book
-:license: Apache License v2.0
+:license: Eclipse Foundation Specification License v1.0
```
Comment (andymc12) [inline full/spec/src/main/asciidoc/index.asciidoc:26]: Same question here.

Comment (phillip-kruger) [reply full/spec/src/main/asciidoc/index.asciidoc:26]: I basically done what has been done and approved in OpenAPI 2.0. But happy to also change the headers

Comment (andymc12) [reply full/spec/src/main/asciidoc/index.asciidoc:26]: Works for me.  :)

**Review by andymc12 — APPROVED** [review]

**Pull request merged**